tests/unittests/analyze/test_boot.py

```diff
@@ -181,3 +183,65 @@ def test_container_ci_log_line(self, m_is_container, m_subp, m_get, m_g):
 
         self.remove_dummy_file(path, log_path)
         assert CONTAINER_CODE == finish_code
+
+    @mock.patch("time.time", return_value=100)
+    @mock.patch("cloudinit.util.uptime", return_value=96)
+    @mock.patch("time.monotonic", return_value=97)
+    @mock.patch("cloudinit.analyze.show.SystemctlReader")
+    @pytest.mark.parametrize(
+        (
+            "is_container_returnvalue",
+            "time_mock_expected_callcount",
+            "monotonic_mock_expected_callcount",
+            "expected_return",
+        ),
+        (
+            (True, 2, 1, (CONTAINER_CODE, 4, 33, 43)),
+            (False, 1, 0, (SUCCESS_CODE, 4, 34, 44)),
+        ),
+    )
+    def test_gather_timestamps_using_systemd(
```

Comment:
I am not directly testing my new (private) function `cloudinit.analyze.show.__get_baselines()` but rather the only function calling it: `cloudinit.analyze.show.gather_timestamps_using_systemd()` which already existed but didn't have unit tests